src/StreamJsonRpc.Desktop/MultilingualResources/StreamJsonRpc.Desktop.cs.xlf

```diff
@@ -148,6 +148,10 @@
           <source>This operation is not allowed before listening for messages has started.</source>
           <target state="translated">Před zahájením naslouchání zprávám není tato operace povolená.</target>
         </trans-unit>
+        <trans-unit id="ParameterNotObject" translate="yes" xml:space="preserve">
+          <source>Parameter is not in the form of a single object</source>
+          <target state="new">Parameter is not in the form of a single object</target>
```

Comment:
I guess you have the MAT extension installed in your VS, is that right? I would have expected it to make this change. Translation comes later. What you have should be fine, so long as it's what MAT did.